improver_tests/calibration/dataframe_utilities/test_dataframe_utilities.py

```diff
@@ -165,19 +165,36 @@ def setUp(self):
             np.array(self.height, dtype=np.float32), "height", units="m",
         )
 
-        self.forecast_df_station_id = self.forecast_df.copy()
-        self.forecast_df_station_id = self.forecast_df_station_id.loc[
-            self.forecast_df_station_id["wmo_id"].isin(self.wmo_ids[:-1])
+        df = self.forecast_df
```

Comment:
I think this section could do with some comments to explain what kinds of dataframes you're trying to set up.

Reply:
Thanks. I've now added a comment.